source/spawn/spawn-posix.ads

```diff
@@ -123,12 +123,8 @@ package Spawn.Posix is
    function fcntl
      (fd    : Interfaces.C.int;
       cmd   : Interfaces.C.int;
-      flags : Interfaces.C.int;
-      dummy : Interfaces.C.C_float := 0.0)
-        return Interfaces.C.int
-          with Import, Convention => C, External_Name => "fcntl";
-   --  An extra float argument is used to make this binding compatible
-   --  with amd64 ABI for C functions with ellipsis (...).
+      flags : Interfaces.C.int) return Interfaces.C.int
```

Comment:
Ada 2022 has a [dedicated Convention for variadic C function.](https://learn.adacore.com/courses/whats-new-in-ada-2022/chapters/importing_variadic_functions.html) Could we use it instead of C wrapper?

Reply:
I prefer C wrapper :)